src/features/projects/styles/ProjectsMap.module.scss

```diff
@@ -738,10 +738,6 @@
   background-color: #828282;
 }
 
-.conservationMarker {
```

Comment:
This style is still used at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/features/projects/components/maps/Location.tsx#L31

Reply:
@mariahosfeld do I need to change markers used here as well?